docs/app/parent-child.adoc

```diff
@@ -181,3 +181,11 @@ parent:
     - RUNNER_STOPPED
     - RUNNER_REMOVED
 ----
+
+=== Filtering Events Based on Commands
+There is a Command Publishing Blocklist that allows commands to be added to the list through the UI. Any events involving
+requests for commands in the blocklist will be filtered out and not sent to the parent.
+
+An issue with adding a command to the list will make any request for that command created on the parent garden and sent
```

Comment:
I think you should take another crack at this wording and also add the `WARNING:` prefix so that it stands out more.  Here's my stab at it:

>WARNING: Once a command has been added to the blocklist, tasking status will no longer be available on the parent garden. If a blocked command is tasked through the parent garden, the request will still be forwarded to the child and executed.  However, the parent will always show a status of CREATED, as the child will no longer be publishing task updates back up to the parent.


Feel free to tweak that if it doesn't quite capture what you were going for.